src/sendCommercialMetrics.ts

```diff
@@ -23,54 +23,71 @@ type EventProperties = {
 };
 
 type CommercialMetricsPayload = {
-	page_view_id: string | null;
-	browser_id?: string | null;
-	platform: 'NEXT_GEN' | null;
-	metrics: readonly Metric[] | null;
-	properties: readonly Property[] | null;
+	page_view_id?: string;
+	browser_id?: string;
+	platform?: 'NEXT_GEN';
+	metrics?: readonly Metric[];
+	properties?: readonly Property[];
 };
 
 enum Endpoints {
 	CODE = '//performance-events.code.dev-guardianapis.com/commercial-metrics',
 	PROD = '//performance-events.guardianapis.com/commercial-metrics',
 }
 
-const commercialMetricsPayload: CommercialMetricsPayload = {
-	page_view_id: null,
-	browser_id: null,
+let commercialMetricsPayload: CommercialMetricsPayload = {
```

Comment:
A `const` can still have its keys mutated. You might be able to keep it as such. 

Not a blocker, but maybe we can omit the `metrics` and `properties` keys here, as they will only ever be populated once and have the metrics be a parameter of `sendMetrics`? In coreWebVitals, we were constrained by the inner workings of Google’s `web-vitals` package, but here we can strive for a pure approach.

Reply:
There's an update to the whole object in the reset, so I switched it to let here.